pkgs/development/python-modules/ecos/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+  # check inputs
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "ecos";
+  version = "2.0.7.post1";
```

Comment:
we should avoid dev releases, if there's an import patch, then I would rather that be facilitated through `patches` and `fetchpatch` of the commits

Reply:
https://github.com/embotech/ecos-python/#what-happened-to-207

> Because version-syncing ECOS and ECOS-Python can be tricky, the 2.0.7 version did not incorporate some minor changes to ECOS. In an ill-advised move, the release was deleted in hopes it could be re-uploaded, despite plenty warnings stating otherwise.
> 
> Instead, a post release has been made that contains identical content to the 2.0.7 release. Generally, pip should pick up the post release for 2.0.7 and any dependencies such as pip install "ecos>=2.0.5" should still work as expected.

Reply:
So this is actually 2.0.7, just an error on the developers' part when ``twine`` uploading/tagging that release.

Reply:
... the correct thing would have been to do so a 2.0.8. That's what patch releases are for... but ok.